README.md

```diff
@@ -4,14 +4,17 @@ Inspired by the [it takes a village](https://verifa.io/blog/the-verifa-community
 
 ## Why Tabete exists
 
-We’re a circle of hands-on consultants who knows that sharing time together will make us better at our craft.
+We're a circle of hands-on consultants who know that sharing time together will make us better at our craft.
 Having done regular check-ups with other consultants for many years, we've noticed a couple of things:
 
 **Osmosis is real.**<br/>
-Simply being around a group of peers lets ideas, perspectives and knowledge cross-pollinate. The conversations in our meetings or when one shows something they’ve done for their client aren’t just theoretical; they spark inspiration and encourage us to explore how to implement those ideas in our clients' environments.
+
```

Comment:
This extra space between fake header and body, should we have it on all pages then? Also, `<br>` can probably be removed then.

Reply:
I added it because what followed was a list ([ref](https://ocular-d.github.io/styleguide-markdown/lists.html#empty-lines-before-and-after)).
Don't actually have a formatter plugin set up, maybe I should.
(And add some linting to the pipeline while I'm at it.)

Generally, use of `<br />` is something I'd like to avoid, feels a bit hacky.
Maybe turn them into actual headers (`####`)?

We should probably look into setting up a `staging.tabete.se` or have some way of rendering it locally to see the effects of our staging.

Reply:
do a make serve, that'll do it for you

Reply:
but I tried actual headers and atual lists (with -), but I didn't like how it looked and after some time doing css I realised I really liked this way